ventas/models.py

```diff
@@ -0,0 +1,23 @@
+from django.db import models
+from django.db.models.fields import DateTimeField
+
+class Producto(models.Model):
+    
+    Categorias = models.TextChoices('Categoria', 'Lentes Armazones Accesorios Ortóptica Otras')
```

Comment:
Podrías definir los TextChoices así? Creo que de esta forma es más díficil equivocarse en la definición de una nueva categoría.

```suggestion
class Categorias(models.TextChoices):
    LENTES = 'Lentes'
    ARMAZONES = 'Armazones'
    ACCESORIOS = 'Accesorios'
    ORTOPTICA = 'Ortóptica'
    OTRAS = 'Otras'
```

PD: No sabia que de esta forma también funcionaba jaja